composer.json

```diff
@@ -22,10 +22,10 @@
   ],
   "require": {
     "php": ">=5.6",
-    "treehouselabs/serialization": "1.0.x-dev"
+    "treehouselabs/serialization": "1.0"
```

Comment:
2.0 requires php ^7.0

First I merge this one and tag it 1.0, then prepare for 2.0 release.